packages/core/src/accordion/accordion-panel.stories.ts

```diff
@@ -38,78 +37,3 @@ export const API = (args: any) => {
     </cds-accordion>
   `;
 };
```

Comment:
I figured out how to overload the storybook stories allowing us to combine stories into a single mdx file and keep the API table docs that it generates. This will help clean up some of the forms as well in a followup PR.